src/test/java/io/vertx/ext/asyncsql/MySQLClientTest.java

```diff
@@ -95,6 +95,22 @@ public void testInsertedIds(TestContext context) {
       });
     });
   }
+  
+  @Test
```

Comment:
Can you move this test to the tests base class, so PostgreSQL gets the same test?
I guess that will also need to change the `MySQLException` inside of this test.